webapi/src/main/scala/org/knora/webapi/slice/admin/domain/model/DefaultObjectAccessPermissionRepo.scala

```diff
@@ -6,17 +6,44 @@
 package org.knora.webapi.slice.admin.domain.model
 
 import zio.Chunk
+import zio.NonEmptyChunk
 import zio.Task
 
 import org.knora.webapi.slice.admin.domain.model.KnoraProject.ProjectIri
 import org.knora.webapi.slice.admin.repo.service.EntityWithId
 import org.knora.webapi.slice.common.repo.service.CrudRepository
+import org.knora.webapi.slice.resourceinfo.domain.InternalIri
 
 final case class DefaultObjectAccessPermission(
   id: PermissionIri,
   forProject: ProjectIri,
+  forWhat: ForWhat,
+  permission: Chunk[DefaultObjectAccessPermissionPart],
 ) extends EntityWithId[PermissionIri]
 
+enum ForWhat {
```

Comment:
Is this under package ...domain.model namespace? In that case the ForWhat is a little unclear. I know I'm not very familiar with the permissions system, but I would still say it looks too plain.

Reply:
That is true, thanks for pointing that out.

Hence, I have moved the respective properties of the `DefaultObjectAccessPermission` into the companion object for a better name spacing.

=> 74a3227bc29c36e42fe8306ace03504fb9b154a0